cmd/root.go

```diff
@@ -99,11 +99,11 @@ Welcome to the Astro CLI, the modern command line interface for data orchestrati
 		newRunCommand(),
 	)
 
-	if config.CFG.SQLCLI.GetBool() {
-		rootCmd.AddCommand(
-			sql.NewFlowCommand(),
-		)
-	}
+	// if config.CFG.SQLCLI.GetBool() {
```

Comment:
@tatiana lemme know what you want to do here to support software. Should we remove this?

Reply:
Fixed